### DISCUSSION TITLE
How to delete?

### DISCUSSION THREAD

**Discussion by wireinet** [starship/starship#5538] [Q&A] [answered]
Hello! in my case i dont use it and decide to remove. After uninstall from windows i find a error in in terminal мы code:
& : Имя "starship" не распознано как имя командлета, функции, файла сценария или выполняемой программы. Проверьте правильность написания имени, а также наличие и правильность пути, после чего повторите попытку.
C:\Windows\System32\WindowsPowerShell\v1.0\profile.ps1:8 знак:21
+ Invoke-Expression (&starship init powershell)
+                     ~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (starship:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException"

**Comment by davidkna** [comment, marked as answer]
You also need to remove the starship line you added from your PowerShell `$PROFILE`.
